softdev.bib

```diff
@@ -2524,3 +2524,13 @@ @article{aycock03brief
     doi = {10.1145/857076.857077},
     acmid = {857077},
 }
+
+@inproceedings{deutsch84efficient,
+    author = "L. Peter Deutsch and Allan M. Schiffman",
```

Comment:
Should the first author's first name be in full?

Reply:
That's the form he uses on the paper itself, so I think this is correct.